### PR TITLE
[Content] Black Belt Story

### PR DESCRIPTION
**Pull request by acblackford**
## Why are you creating this Pull Request?

- [Adding Datasets or Stories](?title=Content%3A%20%3Cname%3E&expand=1&template=content.md)
- [Version Release](?title=Deploy%20vX.X.X&expand=1&template=version_release.md)
- [Other](?expand=1&template=default.md)

### CODE REVIEW

**Comment by acblackford** [comment]
@anayeaye @slesaad the soil texture dataset in this PR is ready for prod

File: stories/black-belt-climate-ej.stories.mdx [hunk 1]
```diff
@@ -296,10 +266,9 @@ taxonomy:
      * [Census Bureau Data | Occupation](https://www.census.gov/topics/employment/industry-occupation.html)]
     </p>
     <p>
-     **Editor**: Maheshwari Neelam and Derek Koehl; 
+     **Editors**: Maheshwari Neelam and Derek Koehl; 
      **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus ;
```
Comment (anayeaye) [inline stories/black-belt-climate-ej.stories.mdx:270]:
```suggestion
     **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus;
```

**Review by anayeaye — COMMENTED** [review]
remove space and try trigger deploy preview

**Comment by slesaad** [comment]
this image is looking too small now
image

also, for the line charts used, using the `<Chart>` component will be better than using images

File: stories/black-belt-climate-ej.stories.mdx [hunk 2]
```diff
@@ -296,10 +266,9 @@ taxonomy:
      * [Census Bureau Data | Occupation](https://www.census.gov/topics/employment/industry-occupation.html)]
     </p>
     <p>
-     **Editor**: Maheshwari Neelam and Derek Koehl; 
-     **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus ;
+     **Editors**: Maheshwari Neelam and Derek Koehl; 
+     **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus;
      **Science and Content Contributors**: Maheshwari Neelam and Udaysankar Nair;
```
Comment (slesaad) [inline stories/black-belt-climate-ej.stories.mdx:271]:
```suggestion
     **Editors**: Maheshwari Neelam and Derek Koehl;
     
     **Developers**: Andrew Blackford, Trent Cowan, Jerika Christman, Brian Freitag, and Aaron Kaulfus;
     
     **Science and Content Contributors**: Maheshwari Neelam and Udaysankar Nair;
```

if you want them to be on separate lines (which I think will look better), do this ☝️

File: stories/black-belt-climate-ej.stories.mdx [hunk 3]
```diff
@@ -259,18 +228,19 @@ taxonomy:
...
-    <p>
-     Additionally, outdoor occupations such as transportation and construction have become crucial sources of income and have shown growth over the past few years. Data from the U.S. Census Bureau, which examines the number of establishments by employment category in counties like Choctaw and Butler, underscores the economic importance of these sectors.
-    </p>
-  </Prose>
+<Block style={{justifyContent: 'center', alignItems: 'center', margin: '0 auto', width: '80%' }}>
   <Figure style={{ flex: '0 0 auto'}}>
     <Image
       src={new URL('./BB_Estab_AL.png', import.meta.url).href}
       style={{ maxWidth: '800px', height: 'auto' }}
     />
```
Comment (slesaad) [inline stories/black-belt-climate-ej.stories.mdx:236]: these images are blurry, maybe use the [`<Chart>` component](https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/content/MDX_BLOCKS.md#chart)?

**Review by mahineelam — COMMENTED** [review]
edits to text

**Comment by jerikac** [comment]
Hey Code Owners - this PR is ready for review!

**Review by aboydnw — APPROVED** [review]
@acblackford merge when ready

@hanbyul-here @snmln let's include this in the release today if possible

**Comment by aboydnw** [comment]
Actually, nevermind @hanbyul-here @snmln I think this story is still dependent on this ticket: https://github.com/NASA-IMPACT/veda-backend/issues/441

**Comment by jerikac** [comment]
Now that [#441 ](https://github.com/NASA-IMPACT/veda-backend/issues/441) is closed, can we push this story to Prod? @aboydnw

**Comment by aboydnw** [comment]
@mahineelam looks like you've been making the bulk of the changes. when you're ready go ahead and merge.

I just did another quick smoke test and looks good. If you merge here then we can plan to release to production the week of Nov 18th

**Comment by aboydnw** [comment]
@mahineelam based on that proposed release to production then you might want to change the published date before merging

**Review by dzole0311 — APPROVED** [review]

**Pull request merged**